sdk/core/Azure.Core/tests/RetriableStreamTests.cs

```diff
@@ -371,16 +371,20 @@ private Task<int> ReadAsync(Stream stream, byte[] buffer, int offset, int length
         private static Stream SendTestRequest(HttpPipeline pipeline, long offset)
         {
             using Request request = CreateRequest(pipeline, offset);
-
-            Response response = pipeline.SendRequest(request, CancellationToken.None);
+            HttpMessage message = new HttpMessage(request, ResponseClassifier.Shared);
+            message.BufferResponse = false;
+            pipeline.Send(message, CancellationToken.None);
+            Response response = message.Response;
             return response.ContentStream;
         }
 
         private static async ValueTask<Stream> SendTestRequestAsync(HttpPipeline pipeline, long offset)
         {
             using Request request = CreateRequest(pipeline, offset);
-
-            Response response = await pipeline.SendRequestAsync(request, CancellationToken.None);
+            HttpMessage message = new HttpMessage(request, ResponseClassifier.Shared);
```

Comment:
I think we can just delete this type.

Reply:
Actually, we can't - it is used by Storage.  It might be a good idea to move it to Storage.Common.